prime-router/metadata/hl7_mapping/resources/Organization/XON.yml

```diff
@@ -13,16 +13,12 @@ elements:
       cweField: '%{hl7XONField}-2'
     schema: ../../datatypes/codeableConcept/CWE
 
-  - name: id-number
-    value: [ '%resource.extension(%`rsext-xon3-id-number`).value' ]
-    hl7Spec: [ '%{hl7XONField}-3' ]
-
   - name: identifier-check-digit
-    value: [ '%resource.identifier.extension(%`rsext-check-digit`).value' ]
+    value: [ '%resource.identifier.extension(%`ext-identifier-checkDigit`).value' ]
```

Comment:
I see that this isn't new, but I'm not familiar with it, and I can't see quickly where `ext` is defined or why this works, do you happen to know?

Reply:
This is actually baked into the fhir path library itself (I assume thats where we got the inspiration from)